### PR TITLE
github actions

### PR DESCRIPTION
**Pull request by benjlevesque**
As stated in #392, travis build is not working and should be replaced with github actions.

I had to tweak the Karma config, without this change the build is failing with this error:
```
19 03 2021 12:25:43.829:ERROR [launcher]: Cannot start Chromium
	xdg-settings: unknown desktop environment
/bin/bash: /etc/profile.d/env_vars.sh: Permission denied
mkdir: cannot create directory ‘/run/user/1001’: Permission denied
No protocol specified
[3426:3426:0319/122543.713023:ERROR:browser_main_loop.cc(1390)] Unable to open X display.
```

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,28 +0,0 @@
...
-addons:
-  chrome: stable
-  firefox: stable
-
-script:
-- echo "versions"
-- openssl version
-- node -v
-- npm install
-- npm run lint
```
Comment (pubkey) [inline .travis.yml:21]: @benjlevesque shouldnt the linting also be in the github actions commands?

Comment (benjlevesque) [reply .travis.yml:21]: oops, yeah I missed that! will add it 👍

Comment (benjlevesque) [reply .travis.yml:21]: @pubkey fixed!

**Comment by pubkey** [comment]
Great. Merged. thank you very much

**Pull request merged**